src/EA.Weee.Web/Areas/Aatf/Views/ManageEvidenceNotes/_EditEvidenceNotePartial.cshtml

```diff
@@ -102,7 +102,7 @@
                     Provide tonnage to 3 decimal places. For example 1.238
                 </p>
             </div>
-            <div class="govuk-grid-column-one-half govuk-body govuk-!-font-size-19 govuk-!-padding-top-5" style="padding-left: 1em">
+            <div class="govuk-grid-column-one-half govuk-body govuk-!-font-size-19 govuk-!-padding-top-5 govuk-!-padding-right-0" style="padding-left: 1em">
```

Comment:
would this be possible by doing govuk-!-padding-left-1 ?

Reply:
1em is 16px. so maybe try govuk-!-padding-left-3?